x/posts/keeper/common_test.go

```diff
@@ -74,12 +64,14 @@ func (suite *KeeperTestsuite) SetupTest() {
 	suite.ctx = sdk.NewContext(ms, tmproto.Header{ChainID: "test-chain"}, false, log.NewNopLogger())
 	suite.cdc, suite.legacyAminoCdc = app.MakeCodecs()
 
+	// Dependencies initializations
 	paramsKeeper := paramskeeper.NewKeeper(suite.cdc, suite.legacyAminoCdc, keys[paramstypes.StoreKey], tKeys[paramstypes.TStoreKey])
-
-	authKeeper := authkeeper.NewAccountKeeper(suite.cdc, keys[authtypes.StoreKey], paramsKeeper.Subspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, app.GetMaccPerms())
 	suite.sk = subspaceskeeper.NewKeeper(suite.cdc, keys[subspacestypes.StoreKey])
```

Comment:
"x/posts highly relies on x/subspaces, so `testutil.MockSubspacesKeeper` cannot be used as a replacement because it requires the use of the non-pure function `IterateSubspaces` within the expected subspaces keeper.